RecoLocalCalo/HcalRecProducers/src/HcalHitReconstructor.cc

```diff
@@ -275,21 +275,13 @@ HcalHitReconstructor::HcalHitReconstructor(edm::ParameterSet const& conf):
               conf.getParameter<int>     ("pedestalSubtractionType"),
               conf.getParameter<double>  ("pedestalUpperLimit"),
               conf.getParameter<int>     ("timeSlewParsType"),
-              conf.getParameter<std::vector<double> >("timeSlewPars")
+              conf.getParameter<std::vector<double> >("timeSlewPars"),
+	      conf.getParameter<double>  ("respCorrM3")
               );
   }
 
 }
 
-void HcalHitReconstructor::fillDescriptions(edm::ConfigurationDescriptions& descriptions) {
-  edm::ParameterSetDescription desc;
-  desc.setAllowAnything();
-  desc.add<int>("pedestalSubtractionType", 1); 
-  desc.add<double>("pedestalUpperLimit", 2.7); 
-  desc.add<int>("timeSlewParsType",3);
-  desc.add<std::vector<double>>("timeSlewPars", {9.27638, -2.05585, 9.27638, -2.05585, 9.27638, -2.05585});
-  descriptions.add("hltHbhereco",desc);
```

Comment:
By the way, you can automatically generate most of the `fillDescriptions()` code from the cfi file itself.

For example,
```
edmPythonConfigToCppValidation RecoLocalCalo/HcalRecProducers/python/HcalHitReconstructor_hbhe_cfi.py
```
will print all the code that you need to implement one of the descriptions.
You can do the same for the other cfi files, and merge the result into a single `fillDescritions()` method.

(yes, I agree that the generated code is not pretty, but I made it look as pretty as possible, to clean it up further would require changing the `edm::ParameterSet` class itself. Anyway, one thing that you can do is to re-format the lists according to their meaning; that should improve the legibility of the code.)